platform/tron/api.go

```diff
@@ -25,17 +25,24 @@ func (p *Platform) Coin() coin.Coin {
 }
 
 func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
-	srcTxs, err := p.client.GetTxsOfAddress(address, "")
+	Txs, err := p.client.GetTxsOfAddress(address, "")
 	if err != nil {
 		return nil, err
 	}
 
+	if len(Txs) == 0 {
+		return nil, err
+	}
+
 	var txs []blockatlas.Tx
-	for _, srcTx := range srcTxs {
-		tx, ok := Normalize(&srcTx)
-		if ok {
-			txs = append(txs, tx)
+	for _, srcTx := range Txs {
+		if len(txs) <= blockatlas.TxPerPage {
```

Comment:
can we just return all for now? not all of them might be normalized. we should not have this logic on each provider, generic one on not, so let's not change this

Reply:
Let's do in separate PR

Reply:
can you remove this change? otherwise lgtm,